docs/source/benchmark.mdx

```diff
@@ -0,0 +1,45 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Benchmarking
+
+## Run
+
+[[autodoc]] optimum.runs_base.Run
```

Comment:
The docs can only find modules that have an init. Since `optimum` cannot have a root init, you'll need to move this to one of the sub-packages

Reply:
Thank you, in the end it was actually just pydantic missing as a dependency!